tests/cli/report-interval.test

```diff
@@ -0,0 +1,60 @@
+
+# Input for the following tests:
+
+2019-01-01
+    (a)   2
+
+2019-02-01
+    (a)   1
+
+
+
+
+# The last report interval option takes precedence.
+
+$ hledger -f- register --weekly --monthly
+>
```

Comment:
Oh, now I understand why you decided to define the order to be `< $ > >2 >=` (vs. `$ < > >2 >=`).

I first preferred the latter form because the command is the most important line an should be on top (in my opinion).

But switching the `<` and `$` lines seems to be OK anyway, shelltest does not throw an error. Is this intended?